app/views/operations/index.html.erb

```diff
@@ -136,22 +99,22 @@
           Usually Scheduled {{current.operation_type.timing.as_string}}
      </div>
 
-    <oplist operationtype="current.operation_type"
+
+    <oplist ng-if="current.operation_type && current.status != 'scheduled' && current.status != 'running'"
+            operationtype="current.operation_type"
             operations="current.operation_type.operations"
-            status="current.status"
-            ng-if="current.operation_type && current.status != 'scheduled' && current.status != 'running'">
+            status="current.status">
     </oplist>
 
-    <div ng-if="current.operation_type && current.status == 'scheduled' || current.status == 'running'">
+    <md-content ng-if="current.operation_type && current.status == 'scheduled' || current.status == 'running'">
 
       <div ng-if="jobs.length > 0" ng-repeat="job_id in jobs" class='job'  style='margin: 16px'>
 
-        <md-toolbar class="job-toolbar">
-
+        <md-toolbar class='job-toolbar'>
           <div class='md-toolbar-tools'>
-
-            <h2 flex md-truncate>
-              Job <a href="/jobs/{{job_id}}">{{job_id}}</a>
+            <h2 flex md-truncate style="color: #eee;">
```

Comment:
if there is any chance that we'll need to tweak this color before replacing the front-end, please use `app/assets/stylesheets/variables.css.scss`